geopandas/array.py

```diff
@@ -795,6 +795,9 @@ def _from_sequence(cls, scalars, dtype=None, copy=False):
         -------
         ExtensionArray
         """
+        # GH 1413
+        if not pd.api.types.is_list_like(scalars):
+            scalars = [scalars]
```

Comment:
Hmm, Multi geometries are still going to fail I think, since:

```
In [23]: pd.api.types.is_list_like(shapely.geometry.MultiPoint())                                                                                                                                                  
Out[23]: True
```

So I think this probably needs something like:

```
is isinstance(scalars, BaseGeometry):
  scalars = [scalars]
```

It should be fine to just check for Shapely geometries, since the scalars you get out of a GeoSeries are always shapely geometries, regardless of whether shapely or pygeos is used under the hood.


It might be good to add a test case for this with something like `pd.Series(MultiPoint, index=[1, 2, 3], dtype="geometry")`

Reply:
> It might be good to add a test case for this with something like `pd.Series(MultiPoint, index=[1, 2, 3], dtype="geometry")`

That does not work at all due to #992, pandas tries to unpack it and raises ValueError at different point. I have changed the check to BaseGeometry, but in practice there is no difference at the moment as multi- will fail anyway.

Reply:
Ah, yes, that indeed then fails in a different place .. 
Anyway, the BaseGeometry check should be more robust if at some point that gets fixed on the pandas side and Multi geometries are actually passed through